__tests__/helpers/requestMock.ts

```diff
@@ -0,0 +1,30 @@
+import { ClientRequest, request } from "http";
+import IntegrationTestEnvironment from "./integrationTestEnvironment";
+
+export default function requestMock(
+  env: IntegrationTestEnvironment,
+  mockID: string,
+  method: "GET" | "POST" = "GET",
+  body?: any,
```

Comment:
can we re-use node hints: `string, Buffer, ArrayBuffer, Array, or array-like object`

Reply:
Not sure I understand...

Reply:
Shall we use explicit types instead of `any`?

Reply:
Ah, yeah! I've already changed to `Buffer | string | any[]` and initialized it with an empty string.